### DISCUSSION TITLE
ServoDyn

### DISCUSSION THREAD

**Discussion by Unique-Wang** [OpenFAST/openfast#908] [Q&A] [answered]
Dear @jjonkman ,

Sorry to bother you again.  lately I've been trying to start a model on openfast, and when I tried to run it an error which is shown in the figure down below occurred. I failed to find the answer from the manual, could you please tell me where went wrong? Thank you!

Best regards,
![123](https://user-images.githubusercontent.com/87693817/139818316-b5d9dde7-cbd2-44f2-a58e-ac5322a09415.png)

**Comment by jjonkman** [comment, marked as answer]
Dear @Unique-Wang,

The first error is related to the mesh-mapping search for ED_2_AD_TowerMotion, which internal to the source code is called `ED_L_2_AD_L_T`, which is the mesh-mapping search between the ElastoDyn Line2 tower motion output mesh and the AeroDyn Line2 tower motion input mesh. I'm not sure why this error is triggered in your case, but my guess is that you have an incompatibility between how the tower discretization is defined in ElastoDyn and AeroDyn. OpenFAST will trigger this error, e.g., if the tower in AeroDyn starts lower than the tower in ElastoDyn or if the tower in AeroDyn ends higher than the tower in ElastoDyn.

Likewise, errors 2-4 are related to the mesh-mapping search for ED_2_AD_BladeMotion, which internal to the source code is called `BDED_L_2_AD_L_B`, which is the mesh-mapping search between ElastoDyn Line2 blade motion output mesh and the AeroDyn Line2 blade motion input mesh.  As with the tower, this is pointing towards an incompatibility between how the blade is discretized in ElastoDyn and AeroDyn.

Best regards,
